pixel-saver@deadalnix.me/app_menu.js

```diff
@@ -176,16 +194,20 @@ function onAppMenuHover(actor) {
 /*
  * Subextension hooks
  */
+let settings;
 function init() {
 	tooltip = new St.Label({
 		style_class: 'tooltip dash-label',
 		text: '',
 	});
+	settings = Convenience.getSettings();
```

Comment:
I don't think exposing the setting format to all the code is a smart move. That would mean changing anything in the setting could break code all over the place. If we are going to have settings, I'd rather keep them encapsulated.